benchmarks/dynamo/common.py

```diff
@@ -3197,16 +3232,28 @@ def detect_and_mark_batch(t):
                     args.per_process_memory_fraction
                 )
 
-            model, example_inputs = runner.cast_based_on_args(model, example_inputs)
-            runner.run_one_model(
-                name,
-                model,
-                example_inputs,
-                optimize_ctx,
-                experiment,
-                explain=args.explain,
-                tag=args.tag,
+            # Set translation validation on by default on CI accuracy runs.
+            ci = CI(
+                args.backend,
+                training=args.training,
+                dynamic=args.dynamic_shapes,
+                device=device,
             )
+            translation_validation = args.only not in CI_TV_OFF[ci]
```

Comment:
This condition is actually incorrect. I should also take into account `args.accuracy` and `args.ci`.